flow/Platform.cpp

```diff
@@ -1412,7 +1412,7 @@ void setMemoryQuota( size_t limit ) {
 	}
 	if (!AssignProcessToJobObject( job, GetCurrentProcess() ))
 		TraceEvent(SevWarn, "FailedToSetMemoryLimit").GetLastError();
-#elif defined(__linux__)
+#elif defined(__linux__) && !defined(USE_ASAN)
```

Comment:
This is no longer necessary as of https://github.com/apple/foundationdb/commit/eacde470500ca0629ff809debcc20473a6f9d4f2